jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -54,7 +54,7 @@ This new organization allows you to manage users and GitLab projects within the
 
 Once you successfully create a new organization, you will be taken to the organization dashboard. Follow the steps in <<#step-two-create-a-new-project,Step Two>> below to set up a new GitLab project.
 
-NOTE: Existing CircleCI users can also opt to sign up via email and password. If you choose to connect a VCS (GitHub or Bitbucket) account that you already have with your new email/password account, your accounts will be merged based on the VCS account name that you used. More information can be found in <<#user-account-integrations,Account integrations>> under the User settings section in this document.
+NOTE: Existing CircleCI users can also opt to sign up with a new email and password account. When you connect the same VCS account to your email/password account, your CircleCI accounts will be merged. More information can be found in <<#user-account-integrations,Account integrations>> under the User settings section in this document.
```

Comment:
@jessiejjacobson @Frozenfire92 I changed the wording a bit. Does this still convey the correct meaning?

Reply:
Yes. I think that is clean and makes the point. @Frozenfire92 agree?

Reply:
Yep, thanks for checking @kbatuigas @jessiejjacobson

Reply:
I guess adding on here.... the phrase "with a new email" makes it seem like CircleCI is going to ask you to create a CircleCI email just for this purpose OR that you are creating a brand new email just for this purpose? Are we trying to say someone can use an existing email with a new password just for CircleCI?

Reply:
@devinmarieb ah, I was thinking more along the lines of, there is the "Sign up with Email" option on the signup page, which I believe _first_ creates a brand new email+password account, and then if you connect the VCS later then that's when the new account gets merged into the existing one. I guess we can also just say "sign up using email and password"